src/java/org/apache/cassandra/tools/nodetool/ListSnapshots.java

```diff
@@ -64,15 +63,17 @@ public void execute(NodeProbe probe)
                 return;
             }
 
-            final long trueSnapshotsSize = probe.trueSnapshotsSize();
             TableBuilder table = new TableBuilder();
             // display column names only once
-            final List<String> indexNames = snapshotDetails.entrySet().iterator().next().getValue().getTabularType().getIndexNames();
+            List<String> indexNames = snapshotDetails.entrySet().iterator().next().getValue().getTabularType().getIndexNames();
+            indexNames.subList(0, indexNames.size() - 1);
```

Comment:
the logic below deals this the fact that we do not want to display "Raw true size" (or ... do we?) in nodetool status list.

I think it makes perfect sense to show it too, but since we are very cautious about how the output looks like, we are not displaying that.

It might be exposed by additional argument as done in case of ephemeral snapshots (`-e` flag), here we would introduced `-t` flag for "true size" or similar.